library/choose_disk.py

```diff
@@ -0,0 +1,436 @@
+#!/usr/bin/python
+
+from ansible.module_utils.basic import AnsibleModule
+import json
+import logging
+import os
+import re
+import subprocess
+
+DOCUMENTATION = '''
+---
+module: choose_disk
+author: Erwan Velu <erwan@redhat.com>
+short_description: Choose disks based on their features
+description:
+    To be completed
+'''
+
+logger = logging.getLogger('choose_disk')
+module = None
+
+
+def _equal(left, right):
+    return left == right
+
+
+def _gt(left, right):
+    return float(left) > float(right)
+
+
+def _lt(left, right):
+    return float(left) < float(right)
+
+
+_REGEXP = re.compile(r'^([^(]+)'          # function name
```

Comment:
that have the benefits of being extandable and support stuff like between(x,y). The real call is lt() or gt() the underscore here is to avoid any overloading between a potential python function and our 'language'.